test/e2e/functional/tests/e2e.go

```diff
@@ -354,6 +365,826 @@ var _ = Describe("Ingress Node Firewall", func() {
 		})
 	})
 
+	Context("[TCP] Rules", func() {
+		var (
+			client                               *corev1.Pod
+			server                               *corev1.Pod
+			config                               *ingressnodefwv1alpha1.IngressNodeFirewallConfig
+			inf                                  *ingressnodefwv1alpha1.IngressNodeFirewall
+			err                                  error
+			serverPodName                        = "e2e-inf-rules-server-"
+			serverListenPort                     = "80"
+			additionalServerPodName              = "e2e-inf-rules-additional-server-"
+			additionalAdditionalServerPodName    = "e2e-inf-rules-additional-additional-server-"
+			additionalAdditionalServerListenPort = "8080"
```

Comment:
:) do u need to put loop for additional , can u use "server-pod1" server-pod2" ?

Reply:
yeah, wasn't a good naming scheme. Renamed it to your suggestion minus the hyphen.